libopenage/renderer/opengl/geometry.cpp

```diff
@@ -54,6 +54,7 @@ void GlGeometry::update_verts_offset(std::vector<uint8_t> const &verts, size_t o
 void GlGeometry::draw() const {
 	switch (this->get_type()) {
 	case geometry_t::bufferless_quad:
+		// a VAO must be bound before this call. This is handled by GlRenderer.
```

Comment:
references to other classes outdated quickly when the code is refactored. Better just state what needs to happen.

```suggestion
		// any VAO must be bound before this call
```